### PR TITLE
Fixed add navigator component

### PR DESCRIPTION
**Pull request by JakubBrozek**
# Tasks
- make new `Navigation` component

# Steps
- [x] active links added
- [x] header added
- [x] subpages added

# Preview
image

### CODE REVIEW

File: src/pages/news.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import Image from "../components/image"
+import SEO from "../components/seo"
+import Navigation from '../components/navigation'
+
+const StyledWrapper = styled.div`
+  background-color: white;
+  color: #404040;
+` ;
+
+
```
Comment (wozny1989) [inline src/pages/news.js:21]: Remove one empty line ✂️

File: src/pages/galery.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import Image from "../components/image"
+import SEO from "../components/seo"
+import Navigation from '../components/navigation'
+
+const StyledWrapper = styled.div`
+  background-color: white;
+  color: #404040;
+` ;
+
+
```
Comment (wozny1989) [inline src/pages/galery.js:21]: Remove one empty line ✂️

File: src/components/header.js [hunk 3]
```diff
@@ -1,42 +1,13 @@
 import { Link } from "gatsby"
 import PropTypes from "prop-types"
 import React from "react"
```
Comment (wozny1989) [inline src/components/header.js]: Remove unused imports ✂️

File: src/pages/about.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Link } from "gatsby"
```
Comment (wozny1989) [inline src/pages/about.js:2]: Remove unused imports ✂️

File: src/pages/about.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+import styled from 'styled-components'
+import Header from '../components/header'
+import Layout from "../components/layout"
+import Image from "../components/image"
+import SEO from "../components/seo"
```
Comment (wozny1989) [inline src/pages/about.js]: Remove unused imports ✂️

File: src/pages/contact.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Link } from "gatsby"
```
Comment (wozny1989) [inline src/pages/contact.js:2]: Remove unused imports ✂️

File: src/pages/contact.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+import styled from 'styled-components'
+import Header from '../components/header'
+import Layout from "../components/layout"
+import Image from "../components/image"
+import SEO from "../components/seo"
```
Comment (wozny1989) [inline src/pages/contact.js]: Remove unused imports ✂️

File: src/pages/courses.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Link } from "gatsby"
```
Comment (wozny1989) [inline src/pages/courses.js:2]: Remove unused imports ✂️

File: src/pages/courses.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+import styled from 'styled-components'
+import Header from '../components/header'
+import Layout from "../components/layout"
+import Image from "../components/image"
+import SEO from "../components/seo"
```
Comment (wozny1989) [inline src/pages/courses.js]: Remove unused imports ✂️

File: src/pages/galery.js [hunk 10]
```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Link } from "gatsby"
```
Comment (wozny1989) [inline src/pages/galery.js:2]: Remove unused imports ✂️

File: src/pages/galery.js [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+import styled from 'styled-components'
+import Header from '../components/header'
+import Layout from "../components/layout"
+import Image from "../components/image"
+import SEO from "../components/seo"
```
Comment (wozny1989) [inline src/pages/galery.js]: Remove unused imports ✂️

File: src/pages/index.js [hunk 12]
```diff
@@ -1,22 +1,24 @@
 import React from "react"
 import { Link } from "gatsby"
```
Comment (wozny1989) [inline src/pages/index.js:2]: Remove unused imports ✂️

File: src/pages/index.js [hunk 13]
```diff
@@ -1,22 +1,24 @@
...
-
+import styled from 'styled-components'
+import Header from '../components/header'
 import Layout from "../components/layout"
 import Image from "../components/image"
 import SEO from "../components/seo"
```
Comment (wozny1989) [inline src/pages/index.js]: Remove unused imports ✂️

File: src/pages/news.js [hunk 14]
```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Link } from "gatsby"
```
Comment (wozny1989) [inline src/pages/news.js:2]: Remove unused imports ✂️

File: src/pages/news.js [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+import styled from 'styled-components'
+import Header from '../components/header'
+import Layout from "../components/layout"
+import Image from "../components/image"
+import SEO from "../components/seo"
```
Comment (wozny1989) [inline src/pages/news.js]: Remove unused imports ✂️

File: src/pages/courses.js [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+import Navigation from '../components/navigation'
+
+const StyledWrapper = styled.div`
+  background-color: white;
+  color: #404040;
+` ;
+
+
```
Comment (wozny1989) [inline src/pages/courses.js:12]: Remove one empty line ✂️

File: src/pages/contact.js [hunk 17]
```diff
@@ -0,0 +1,25 @@
...
+import Navigation from '../components/navigation'
+
+const StyledWrapper = styled.div`
+  background-color: white;
+  color: #404040;
+` ;
+
+
```
Comment (wozny1989) [inline src/pages/contact.js:21]: Remove one empty line ✂️

File: src/pages/about.js [hunk 18]
```diff
@@ -0,0 +1,25 @@
...
+import Navigation from '../components/navigation'
+
+const StyledWrapper = styled.div`
+  background-color: white;
+  color: #404040;
+` ;
+
+
```
Comment (wozny1989) [inline src/pages/about.js:21]: Remove one empty line ✂️

File: src/components/navigation.js [hunk 19]
```diff
@@ -0,0 +1,62 @@
...
+
+const StyledLink = styled(Link)`
+    color: #404040;
+    text-decoration: none;
+    
+    &.active{
+        color: orange;
+        font-weight: 600;
+    }
+
```
Comment (wozny1989) [inline src/components/navigation.js:38]: Remove empty line ✂️

File: src/components/navigation.js [hunk 20]
```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import styled from 'styled-components'
+import {Link} from 'gatsby'
+
+
```
Comment (wozny1989) [inline src/components/navigation.js]: Remove empty line ✂️

File: gatsby-config.js [hunk 21]
```diff
@@ -12,6 +12,10 @@ module.exports = {
         name: `images`,
         path: `${__dirname}/src/images`,
       },
+      resolve: `gatsby-plugin-styled-components`,
+      options: {
+      
+      },
```
Comment (wozny1989) [inline gatsby-config.js]: Remove unnecessary empty `options`

**Review by wozny1989 — CHANGES_REQUESTED** [review]
@JakubBrozek I left some feedback

File: src/pages/about.js (hunk 4 above)
Comment (JakubBrozek) [reply src/pages/about.js:2]: Those are used

Comment (wozny1989) [reply src/pages/about.js:2]: my fault 👍

**Review by wozny1989 — APPROVED** [review]

**Pull request merged**